### PR TITLE
DASH-442 Add ability to permanently delete user

### PR DESCRIPTION
**Pull request by orkhan-huseyn**

### CODE REVIEW

File: client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import DangerModal from '../../../../components/Modal/DangerModal';
+import Modal from '../../../../components/Modal/Modal';
+import SecurityItem from '../SecurityItem';
+import PasswordForm, { PasswordFormValues } from './PasswordForm';
+
+import { ROUTES } from '../../../../constants/routes';
+
+import apis from '../../../../api';
+
+import classes from '../../styles/DeleteMyAccount.module.scss';
```
Comment (trukhilio) [inline client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx:14]: Move classes into the bottom of imports

File: client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      setCheckingPassword(false);
+    }
+
+    setDeletingAccount(true);
+    try {
+      await apis.auth.deleteUser();
+      await apis.edge.deleteAccount(account);
+      togglePasswordModal(false);
+      toggleSuccessModal(true);
+      setTimeout(logout, 1000);
```
Comment (trukhilio) [inline client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx:82]: Move timeout value into constants

**Review by trukhilio — APPROVED** [review]
Check comment

File: client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx (hunk 1 above)
Comment (orkhan-huseyn) [reply client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx:15]: done

File: client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx (hunk 2 above)
Comment (orkhan-huseyn) [reply client/src/pages/SettingsPage/components/DeleteMyAccount/DeleteMyAccount.tsx]: done

**Pull request merged**